pkg/lzma/lzma.go

```diff
@@ -0,0 +1,46 @@
+// Package lzma implements reading and writing of LZMA compressed files.
+//
+// This package is specifically designed for the LZMA format used popular UEFI
+// implementations. It requires the `lzma` and `unlzma` programs to be
+// installed and on the path.
+package lzma
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+const (
+	lzmaEncode = "lzma"
```

Comment:
make these passable by flag?

Reply:
Maybe like this:

```go
// LZMA configures the LZMA binary to use.
type LZMA struct {
  Encoder, Decoder string
}

// Use whatever is in PATH by default.
var Default = LZMA{
  Encoder: "lzma",
  Decoder: "unlzma",
}

func (l LZMA) Decode(...) {
  cmd := exec.Command(l.Decoder) ...
}

// easy default func
func Decode(...) {
  return Default.Decode(...)
}

func NewLZMA(encoder, decoder string) LZMA {
  
}
```

Reply:
Then, a command can pass the flag to NewLZMA, or use Default by default.

Reply:
Thanks, I like this idea! Will do in another PR.

Reply:
#68